### DISCUSSION TITLE
using Telescope instead of netrw in neovim

### DISCUSSION THREAD

**Discussion by azinsharaf** [twpayne/chezmoi#3511] [Ideas]
how can i use telescope instead of netrw in `chezmoi edit` command when the editor has been set to neovim?

**Comment by halostatue** [comment]
That's really a Neovim question that I think is answered [on SO](https://stackoverflow.com/questions/76028722/how-can-i-temporarily-disable-netrw-so-i-can-have-telescope-at-startup) and with the [nvim-telescope/telescope-file-browser.nvim](https://github.com/nvim-telescope/telescope-file-browser.nvim) plugin.

**Reply by azinsharaf** [reply]
thanks for the response. i couldn't make it happen but i found `chezmoi.nvim` neovim plugin that does the same thing i need.
